src/app/account/settings/ExportTransactions/index.tsx

```diff
@@ -0,0 +1,74 @@
+"use client"
```

Comment:
Also notice that we are using lowercase for the folders in all the project, could you please change the folder name to `export-transactions`? Thanks
<img width="270" alt="image" src="https://github.com/user-attachments/assets/20f7c01b-fd74-47d6-939e-7e8d0a553d5d">